patch-package
--- a/node_modules/record-node/components/feed.js
+++ b/node_modules/record-node/components/feed.js
@@ -25,11 +25,11 @@ module.exports = function feed (self) {
         const { type } = feedEntry.payload
         // TODO: make this readable and less suspect :(
         const entry = await self[`${type}s`].get(address, feedEntry.payload.entryId)
-        entries.push({
-          ...feedEntry.payload,
+        const item = Object.assign({
           contact: contact.payload.value,
           content: entry.payload.value
-        })
+        }, feedEntry.payload)
+        entries.push(item)
       }
       return entries
     }
